src/collections/_documentation/development/contribute/environment.md

```diff
@@ -11,38 +11,114 @@ Follow [the Docker file in the repository](https://github.com/getsentry/sentry/b
 
 ### Macintosh OS X
 
-To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) and clone it:
+To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry). Clone your forked repo and go into it:
 
 ```bash
 git clone https://github.com/<your github username>/sentry.git
 cd sentry
 ```
 
-Install [Homebrew](http://brew.sh), if you haven’t already. Run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`. This will install, among other things, Python 2 and docker.
+{% capture __alert_content -%}
+It's important that you remain in the repo's root directory for the rest of the setup! Also, please do the following steps in the exact order they're shown.
+{%- endcapture -%}{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+
+### Macintosh OS X - System Dependencies
+
+Install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
+
+One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight, search for "Docker" and start it. You should soon see the docker icon in your macOS toolbar. Docker will automatically run on system restarts, so this should be the only time you do this.
+
+You can verify that docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
+
+
+### Macintosh OS X - Python
+
+Unfortunately, homebrew recently removed Python 2. While some versions of macOS ship with Python 2, it's recommended to not use the system's installation. Sentry also requires a specific version of Python, as shown in the file `.python-version`, and this may differ from the system's Python.
```

Comment:
Idea: have and ship prebuilt .venvs that also includes the dependencies from the latest master continuously so "setup" is merely downloading and extracting a tarball?

Reply:
That's a good idea for the future, but the docs need to be fixed _now_. Otherwise we'll have more and more people who are trying to setup sentry for dev running into the same problems.

Reply:
I didn't try to make it sound like it should be implemented _now_. This is not a blocker for this PR.

Reply:
I do not think we can pull this off without dockerizing the entire devenv first. An *installed* binary package is not necessarily compatible across even minor versions of macos or linux